README.md

```diff
@@ -27,6 +27,32 @@ $ npm run dev-server
 
 Open http://localhost:1234 in your browser to see it in action.
 
+## Running in Windows Machine
+
+1. Install copyfiles package 
+
+~~~ console
+$ npm install --save-dev copyfiles to install copyfiles
```

Comment:
I think you can remove "to install ..." so it can't be mistaken as it is in that command !

Reply:
Done removing this